crates/topos-p2p/src/runtime/handle_event/gossipsub.rs

```diff
@@ -9,12 +9,12 @@ use crate::{constants, event::GossipEvent, Event, Runtime, TOPOS_ECHO, TOPOS_GOS
 use prost::Message;
 use topos_core::api::grpc::tce::v1::Batch;
 
-use super::EventHandler;
+use super::{EventHandler, EventResult};
 
 #[async_trait::async_trait]
 impl EventHandler<GossipEvent> for Runtime {
-    async fn handle(&mut self, event: GossipEvent) {
-        if let GossipEvent {
+    async fn handle(&mut self, event: GossipEvent) -> EventResult {
+        if let GossipEvent::Message {
```

Comment:
Do we expect to have more than one variant of `GossipEvent` down the line?

Reply:
Yes we do, mostly to deal with errors